solution_runner/commands/consts.py

```diff
@@ -0,0 +1,32 @@
+from datetime import datetime
+from pathlib import Path
+from zoneinfo import ZoneInfo
+
+import click
+
+
+FIRST_AOC_YEAR = 2015
+ZERO = '0'
+INPUT_URL = 'https://adventofcode.com/{year}/day/{day}/input'
+COOKIE = {'session': None}
```

Comment:
change the name so the usage of that specific cookie will be more clear. COOKIE is too general

Reply:
I ended up deleting this constant because it was not used (eb4acf3).